Modules/Core/Sources/Core/MediaType/MediaType.swift

```diff
@@ -1,7 +1,7 @@
-public protocol MediaTypeRepresentor {
+public protocol MediaTypeConvertible {
```

Comment:
it wasn't `MediaTypeRepresentable` it was `MediaTypeRepresentor`. Yeah I couldn't find a good name. We need a name that can reflect the capability of providing a parser and a serializer. I agree `Convertible` is used consistently with another meaning, but `Representable` is also used consistently with another meaning.